pkg/config/generate.go

```diff
@@ -15,6 +16,18 @@ import (
 	"github.com/openshift/openshift-azure/pkg/tls"
 )
 
+type certificate struct {
```

Comment:
unless you are actually really using this, revert this change

Reply:
It makes `regenerate` function cleaner. I can move it back but this means all logic from regenate goes to main function too...